docs/root/operations/admin.rst

```diff
@@ -174,3 +174,21 @@ The fields are:
   .. http:get:: /runtime?format=json
 
   Outputs /runtime in JSON format. This can be used for programmatic access of runtime values.
+
+.. http:get:: /hystrix_event_stream
+
+  Start streaming :ref:`event stream<https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events/Using_server-sent_events>` to hystrix dashboard. 
```

Comment:
You need to use external RST link syntax rather than internal here, this is what is breaking docs build.